src/services/refactors/addOrRemoveBracesToArrowFunction.ts

```diff
@@ -16,8 +16,8 @@ namespace ts.refactor.addOrRemoveBracesToArrowFunction {
     }
 
     function getAvailableActions(context: RefactorContext): readonly ApplicableRefactorInfo[] {
-        const { file, startPosition } = context;
-        const info = getConvertibleArrowFunctionAtPosition(file, startPosition);
+        const { file, startPosition, triggerReason } = context;
+        const info = getConvertibleArrowFunctionAtPosition(file, startPosition, triggerReason === "invoked");

```

Comment:
Unfortunately, JS doesn't have named arguments.  Instead, when we're passing a boolean, we tend to give the parameter name as a `/*comment*/` before the argument (unless the boolean already has a descriptive name, of course).

Reply:
It wasn't my intention to pass a named argument, but to pass whatever `triggerReason === "invoked"` evaluates to (`false` if `triggerReason` is `"implicit"` or `undefined`. Did I do that incorrectly?